student.rb

```diff
@@ -1,9 +1,12 @@
 require './person'
 
 class Student < Person
+  attr_accessor :classroom
+  
   def initialize(classroom, age, name = "Unknown", parent_permission = true)
     super(age, name, parent_permission)
     @classroom = classroom
+    classroom.students << self
```

Comment:
- [ ] it would be better to remove it from the initialize methods and add a method `classroom` so that you can use it whenever you want to add a student to a classroom right now what if there is no classroom yet that will make an error when creating a student